docs/day-two/quality-of-life-tweaks.md

```diff
@@ -9,7 +9,18 @@ Bash aliases enable complex commands to be shortned to a few characters. For exa
 ```bash
 alias dcp='docker-compose -f ~/docker-compose.yml '
 ```
+### Making aliases persistent
 
+Aliases created using the `alias` command in the terminal will only be available for the current session. To make an alias persist across sessions it must be added the shell's initialization file. Here are some examples for popular shells:
+
+ - Bash: `~/.bashrc`, this is typically the default shell.
+ - ZSH: `~/.zshrc`
+ - Fish: `~/.config/fish/config.fish`
+
+For both Bash and ZSH simply add the desired aliases to the end of the file and reload the file using `. ~/.bashrc` or `. ~/.zshrc`.
+
+Fish is a bit different, the simplest way to save an alias in fish it to add the `-s` or `--save` option to the alias command in the terminal, this will add the alias in the correct format to the `config.fish` file. For example: `alias -s dcp='docker-compose -f ~/docker-compose.yml '`
```

Comment:
I'm not sure this adds anything useful. Might as well have the consistent instructions of "add this line to your shells config" and leave it at that. I assume the syntax in each file is identical for aliases?

Reply:
You are correct. I tend to make things more complicated than they need to be. It is something I am working on 👍